phpcs.xml.dist

```diff
@@ -5,12 +5,318 @@
 
   <arg name="extensions" value="inc,install,module,php,profile,test,theme,yml"/>
 
-  <rule ref="Drupal">
```

Comment:
we need to keep this line, we want to reference the whole Drupal standard first.